vm/actor/tests/reward_actor_test.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+mod common;
+
+use actor::{
+    reward::{AwardBlockRewardParams, Method},
+    REWARD_ACTOR_ADDR, REWARD_ACTOR_CODE_ID, SYSTEM_ACTOR_ADDR, SYSTEM_ACTOR_CODE_ID,
+};
+use address::Address;
+use common::*;
+use db::MemoryDB;
+use ipld_blockstore::BlockStore;
+use message::UnsignedMessage;
+use vm::{ExitCode, Serialized, TokenAmount, METHOD_CONSTRUCTOR};
+
+fn construct_runtime<BS: BlockStore>(bs: &BS) -> MockRuntime<'_, BS> {
+    let message = UnsignedMessage::builder()
+        .to(REWARD_ACTOR_ADDR.clone())
+        .from(SYSTEM_ACTOR_ADDR.clone())
+        .build()
+        .unwrap();
+    let mut rt = MockRuntime::new(bs, message);
+    rt.caller_type = SYSTEM_ACTOR_CODE_ID.clone();
+    return rt;
+}
+
+#[test]
+fn test_award_block_reward() {
```

Comment:
can you rename this to something around what it's doing? (failure when balance is less than reward)